src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="../src/styles/main.scss"
```

Comment:
Link to a SCSS file is incorrect. In an HTML document, you should link to a compiled CSS file, not the SCSS source file. Make sure you compile your SCSS to CSS and link to that CSS file instead.